src/ruby/app/models/keyboard.rb

```diff
@@ -464,6 +464,8 @@ def initialize
     @scan_mode = :matrix
     @skip_positions = Array.new
     @layer_changed_delay = 20
+    @map_size = 0
```

Comment:
@yswallow 
What does `@map_size` work for?
It looks like just being assigned but not being used anywhere.